### PR TITLE
Replaces static text with formatted messages

### PR DESCRIPTION
**Pull request by hermzz**

### CODE REVIEW

File: frontend/modules/base/components/Nav.js [hunk 1]
```diff
@@ -107,27 +106,55 @@ var Account = React.createClass({
...
+      }
+    });
+
     return (
-      <NavDropdown eventKey={1} title="My Account" id="basic-nav-dropdown">
-        <LinkContainer to="/recipe/create"><MenuItem>Create Recipe</MenuItem></LinkContainer>
+      <NavDropdown eventKey={1} title={ formatMessage(messages.title) } id="basic-nav-dropdown">
+        <LinkContainer to="/recipe/create"><MenuItem>{ formatMessage(messages.create_recipe) }</MenuItem></LinkContainer>
         <MenuItem divider />
         <NavItem onClick={this.logout}>Logout</NavItem>
```
Comment (RyanNoelk) [inline frontend/modules/base/components/Nav.js]: You missed the logout here.

File: frontend/modules/create/components/DirectionList.js [hunk 2]
```diff
@@ -34,29 +54,38 @@ export default React.createClass({
...
   updateData: function (id, data) {
-    var tmp = {};
-    tmp['title'] = data;
-    tmp['step'] = id;
     var instructions = this.state.data;
-    instructions[id] = tmp;
+    instructions[id] = {
+      title: data,
+      step: id 
+    };
```
Comment (RyanNoelk) [inline frontend/modules/create/components/DirectionList.js:61]: 👍

**Review by RyanNoelk — CHANGES_REQUESTED** [review]

**Comment by RyanNoelk** [comment]
One small nit, otherwise this looks amazing!!

File: frontend/modules/base/components/Nav.js (hunk 1 above)
Comment (hermzz) [reply frontend/modules/base/components/Nav.js]: Good catch! Fixed.

**Review by RyanNoelk — APPROVED** [review]

**Pull request merged**